machine-learning/pyproject.toml

```diff
@@ -59,6 +59,14 @@ optional = true
 [tool.poetry.group.armnn.dependencies]
 onnxruntime = "^1.15.0"
 
+[tool.poetry.group.cuda12.dependencies]
+onnxruntime-gpu = {version = "^1.17.0", source = "cuda12"}
+
+[[tool.poetry.source]]
+name = "cuda12"
+url = "https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/"
```

Comment:
What is this link lol

Reply:
The official repo 🤷 https://onnxruntime.ai/docs/install/#install-onnx-runtime-gpu-cuda-12x

Reply:
That's the microsoft's devops plateform

Reply:
Hmm fair enough. Weird that they publish it there and not on PyPI